src/components/WorkPoolQueueCreateForm.vue

```diff
@@ -56,7 +56,7 @@
   const routes = useWorkspaceRoutes()
   const { validate, pending } = useValidationObserver()
 
-  const name = ref<string>()
+  const name = ref('')
```

Comment:
`WorkQueueCreate` requires name, but for some reason the ui's WorkPoolQueueCreate and WorkQueueCreate types diverged on this.

https://github.com/PrefectHQ/prefect/blob/6dbf284f9c99b3dfbbecf22638c5d71d621f6b93/src/prefect/server/schemas/actions.py#L845-L846

This form already uses an `isRequired` validation rule here too so shouldn't be any functional change here.